README.org

```diff
@@ -1,32 +1,57 @@
 * boot-cljs-devtools
 ** About
-[[https://github.com/boot-clj/boot][Boot]] task introducing enhancements to CLJS development in Chrome, specifically [[https://github.com/binaryage/cljs-devtools][CLJS DevTools]] and [[https://github.com/binaryage/dirac][Dirac]].
+[[https://github.com/boot-clj/boot][Boot]] tasks introducing enhancements to CLJS development in Chrome, specifically [[https://github.com/binaryage/cljs-devtools][CLJS DevTools]] and [[https://github.com/binaryage/dirac][Dirac]].
 ** Installation
 Latest version:
 
 [[https://clojars.org/powerlaces/boot-cljs-devtools][https://img.shields.io/clojars/v/powerlaces/boot-cljs-devtools.svg]]
 
 In order to install it, add the following to your =build.boot= dependencies:
 #+BEGIN_SRC clojure
-[binaryage/devtools      "0.8.2" :scope "test"]
-[binaryage/dirac         "0.7.1" :scope "test"]
+[binaryage/devtools      "0.8.3" :scope "test"] ;; when you want cljd-devtools or both
+[binaryage/dirac         "0.8.7" :scope "test"] ;; when you want dirac or both
```

Comment:
Agreed, just followed current state. The thing is Dirac is one of the fastest moving targets around 😉